streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -17,116 +17,109 @@
 
 package org.apache.kafka.streams.kstream;
 
+import org.apache.kafka.common.annotation.InterfaceStability;
 import org.apache.kafka.common.serialization.Serde;
 import org.apache.kafka.streams.KeyValue;
 import org.apache.kafka.streams.processor.ProcessorSupplier;
 
 /**
- * KStream is an abstraction of an event stream in key-value pairs.
+ * KStream is an abstraction of a <i>record stream</i> in key-value pairs.
  *
  * @param <K> Type of keys
  * @param <V> Type of values
  */
+@InterfaceStability.Unstable
 public interface KStream<K, V> {
 
     /**
-     * Creates a new instance of KStream consists of all elements of this stream which satisfy a predicate
+     * Create a new instance of {@link KStream} consists of all elements of this stream which satisfy a predicate.
```

Comment:
How are we deciding whether to use `Creates` or `Create`? Looking at this PR, it seems like we use both forms. The style guide for javadoc seems to use the `Creates` form: http://www.oracle.com/technetwork/articles/java/index-137868.html